dataminer-overview/Pricing/Pricing_Usage_based_service.md

```diff
@@ -94,7 +94,7 @@ Consumption above the contracted Monthly Utilization Allowance (MUA) is possible
 | *Standard Managed Object* | Endpoints directly or indirectly interfaced by DataMiner with more than 200 metrics, metered as the count of 10K metrics on each endpoint. Example: A Managed Object with 24K metrics is metered as 3. |
 | *Light Managed Object* | Endpoints with less than 200 metrics, metered as the sum of their metrics. |
 | *Connector Services* | Use of Skyline-developed connectors (also known as DataMiner protocols or interface drivers) made available through the [Catalog](https://catalog.dataminer.services/).<br>Connectors developed by the user or another third party are not counted. |
-| *Script Runs* | Every time Automation scripts are [triggered](xref:Running_Automation_scripts).<br>Amongst others, this includes Life cycle Service Orchestration (LSO) scripts, Profile-Load Scripts (PLS), Process Automation activities, DOM instance state transitions, and user-defined API calls. |
+| *Script Runs* | Every time Automation scripts are [triggered](xref:Running_Automation_scripts).<br>Amongst others, this includes Life cycle Service Orchestration (LSO) scripts, Profile-Load Scripts (PLS), Process Automation activities or DOM instance state transitions.|
```

Comment:
Since this is an enumeration of examples that starts with "Amongst others" the "or" is rather weird. Can this be replaced by "and"?
```suggestion
| *Script Runs* | Every time Automation scripts are [triggered](xref:Running_Automation_scripts).<br>Amongst others, this includes Life cycle Service Orchestration (LSO) scripts, Profile-Load Scripts (PLS), Process Automation activities, and DOM instance state transitions. |
```

Reply:
of course. feel free to correct my grammar anytime you find errors :)